Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageCell.swift

```diff
@@ -65,12 +71,45 @@ public class ChatMessageCell: _TableViewCell, ComponentsProvider {
 
         backgroundColor = .clear
         backgroundView = nil
+
+        headerContainerView.backgroundColor = nil
+        footerContainerView.backgroundColor = nil
     }
 
     override public func prepareForReuse() {
         super.prepareForReuse()
 
         messageContentView?.prepareForReuse()
+        headerContainerView.removeFromSuperview()
+        footerContainerView.removeFromSuperview()
```

Comment:
Do we still want this based on this comment: https://github.com/GetStream/stream-chat-swift/pull/2507#discussion_r1116030956